strutil/pathiter.go

```diff
@@ -85,6 +85,14 @@ func (iter *PathIterator) CurrentBase() string {
 	return iter.path[:iter.left]
 }
 
+// CurrentCleanBase returns the same value as CurrentBase with the right slash trimmed.
+func (iter *PathIterator) CurrentCleanBase() string {
```

Comment:
should we keep CurrentBase() at all, is not used now and seem a nuisance?

Reply:
It can be dropped if there is no compelling reason to keep it.